ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -622,3 +622,34 @@ def test_tensorflow_asinh(
         fn_tree="asinh",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# reduce_variance
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+    ),
+    as_variable=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.reduce_variance"
+    ),
+    native_array=st.booleans(),
+)
+def test_tensorflow_reduce_variance(
+    dtype_and_x, as_variable, num_positional_args, native_array, fw
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="reduce_variance",
```

Comment:
This should be `fn_tree="math.reduce_variance"` because you call the function like `tf.math.reduce_variance`

Reply:
@Unevilicorn I have corrected the `fn_tree` argument. Currently waiting for the checks to be completed.